source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -1349,7 +1349,7 @@ def make_mock_chaos_item(
     }
 
     ptobject = impact.informed_entities.add()
-    ptobject.uri = impacted_obj
+    ptobject.uri = impacted_obj or "None"
```

Comment:
Is it a typo?

```suggestion
    ptobject.uri = impacted_obj or None
```

Reply:
It's not, uri is a pbf object, it's typed.
Not especially proud of what I did on that mock (didn't want to refactor it, but it would be nice).